src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,11 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+namespace Cake\Network\Email;
+
+if(false) {
+    /**
+     * @deprecated Use Cake\Mailer\AbstractTransport instead.
+     */
+    abstract class AbstractTransport extends \Cake\Mailer\AbstractTransport
```

Comment:
If we really need stub classes to make IDE's happy could we have simple class extensions?